src/csharp/Microsoft.Spark/Sql/Functions.cs

```diff
@@ -3797,6 +3798,188 @@ public static Func<Column> Udf<TResult>(Func<TResult> udf)
             return CreateUdf<TResult>(udf.Method.ToString(), UdfUtils.CreateUdfWrapper(udf)).Apply10;
         }
 
+        /// <summary>Creates a UDF from the specified delegate.</summary>
+        /// <param name="udf">The UDF function implementation.</param>
+        /// <param name="returnType">Schema associated with this row</param>
```

Comment:
I wonder if `returnType` isn't the best name for this thing.  The description "Schema associated with this row" wasn't super clear to me, so I had to dig into the code to understand that it's the schema of the row returned by the UDF.

Reply:
I was following `returnType` naming [here](https://github.com/dotnet/spark/blob/7fd0a0315d3935523633fc3e7a258f5e2e63d0eb/src/csharp/Microsoft.Spark/Sql/Expressions/UserDefinedFunction.cs#L20). Do you think `schema` would be better?